docs/src/main/sphinx/release/release-376.md

```diff
@@ -0,0 +1,83 @@
+# Release 376 (7 Apr 2022)
+
+## General
+
+* Add table redirection awareness for `RENAME table` operations. ({issue}`11277`)
+* Deny adding column with comment if the connector does not support this feature. ({issue}`11486`)
+* Improve query plans and performance in the presence of inequality expressions. ({issue}`11518`)
+* Use "is distinct" semantics in `array_except`, `array_union`, `map_concat`,
+  `map_from_entries`, `multimap_from_entries`, and `multimap_agg`. ({issue}`560`)
+* Fix operator type exception that occurred when index join was used during
+  query execution. ({issue}`11758`)
+* Fix stage memory statistics reporting for queries running with `retry-policy`
+  set to `TASK`. ({issue}`11801`)
+
+## Security
+
+* Add support to use two-way TLS/SSL with LDAP authentication. ({issue}`11070`)
+* Fix failures in information schema role tables for catalogs using system roles. ({issue}`11694`)
+
+## JDBC driver
+
+* Remove `user` property requirement. ({issue}`11350`)
+* Add support for `ResultSet.getAsciiStream` and `ResultSet.getBinaryStream`. ({issue}`11753`)
+
+## Accumulo connector
+
+* Support adding and dropping schemas. ({issue}`11808`)
+* Disallow creating tables in a schema where it doesn't exist. ({issue}`11808`)
+
+## ClickHouse connector
+
+* Add support for column comments when creating new tables. ({issue}`11606`)
+* Add support for column comments when adding new columns. ({issue}`11606`)
+
+## Delta Lake connector
+
+* Support views when using Iceberg Glue catalog. ({issue}`11499`)
+* Allow running queries performing DML on Delta tables with fault-tolerant
+  execution. ({issue}`11591`)
+* Allow setting duration for completion of dynamic filters via the
+  `delta.dynamic-filtering.wait-timeout` configuration property. ({issue}`11600`)
+* Improve query planning time after `ALTER TABLE ... EXECUTE optimize` by always
+  creating a transaction log checkpoint during the optimize. ({issue}`11721`)
```

Comment:
can we rephrase to

during the optimization.

@findepi @losipiuk ?

Reply:
Maybe "during the OPTIMIZE procedure"

Reply:
it would have to be lowercase

```
during the `optimize` procedure.
```

but imho thats repetitive to the code snippet earlier in the sentence, means the same as "optimization", and is longer..

Reply:
Fair enough. You could also just end the sentence after "checkpoint"

Reply:
I like that even more ... and since you suggested.. lets roll with that ;-)

Reply:
note: `optimize` is not a procedure, it's a table procedure